server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ProjectFilesController.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.server.web.controller.v2;
+
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.Resource;
+import javax.validation.Valid;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.oceanbase.odc.service.common.response.ListResponse;
+import com.oceanbase.odc.service.common.response.Responses;
+import com.oceanbase.odc.service.common.response.SuccessResponse;
+import com.oceanbase.odc.service.projectfiles.ProjectFilesServiceFacade;
+import com.oceanbase.odc.service.projectfiles.model.BatchUploadProjectFileReq;
+import com.oceanbase.odc.service.projectfiles.model.FileUploadTempCredentialResp;
+import com.oceanbase.odc.service.projectfiles.model.GenerateProjectFileTempCredentialReq;
+import com.oceanbase.odc.service.projectfiles.model.ProjectFileMetaResp;
+import com.oceanbase.odc.service.projectfiles.model.ProjectFileResp;
+import com.oceanbase.odc.service.projectfiles.model.UpdateProjectFileReq;
+
+/**
+ * 项目文件管理控制类
+ *
+ * @author keyangs
+ * @date 2024/7/31
+ * @since 4.3.2
+ */
+@RestController
+@RequestMapping("/api/v2/project/{projectId}/files")
+public class ProjectFilesController {
+
+    @Resource
+    private ProjectFilesServiceFacade projectFilesServiceFacade;
+
+    @PostMapping("/generateTempCredential")
+    public SuccessResponse<FileUploadTempCredentialResp> generateTempCredential(
```

Comment:
for generateTempCredential, we may reuse current API

Reply:
current API only support upload one file. in the context of project files , need to support uploading multiple files，this is incompatible with the current API.
In addition, returning a temporary folder facilitates setting a separate lifecycle. In case of upload interruption or failure, the uploaded files can be automatically deleted after they become invalid.